test/e2e/framework/networking_utils.go

```diff
@@ -218,8 +218,9 @@ func (config *NetworkingTestConfig) DialFromContainer(protocol, containerIP, tar
 // pod and confirm it doesn't show up as an endpoint.
 func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
 	var cmd string
+	// Make the command timeout after 1s (no matter what protocol it is).
 	if protocol == "udp" {
-		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 3 nc -w 1 -u %s %d", targetIP, targetPort)
+		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 1 nc -w 1 -u %s %d", targetIP, targetPort)
```

Comment:
the err logs shows 
```
I1021 01:41:24.014] Oct 21 01:40:32.259: INFO: Waiting for map[netserver-0:{}] endpoints, got endpoints map[]
I1021 01:41:24.014] Oct 21 01:40:32.263: INFO: Exec running '/bin/sh -c echo 'hostName' | timeout -t 1 nc -w 1 -u 10.180.1.89 8081 | grep -v '^\s*$''
I1021 01:41:24.014] Oct 21 01:40:32.263: INFO: >>> kubeConfig: /workspace/.kube/config
I1021 01:41:24.014] 
I1021 01:41:24.015] Oct 21 01:40:33.506: INFO: Failed to execute "echo 'hostName' | timeout -t 1 nc -w 1 -u 10.180.1.89 8081 | grep -v '^\\s*$'": <nil>, stdout: "netserver-0", stderr: "Terminated"
```

So clearly the stdout has the string we're waiting for. But the stderr also has Terminated. This makes me think nc is getting terminated by the timeout command before its own natural termination (-w), maybe this is confusing the test? You should be able to change it to -t 2 and avoid the issue, if that's the cause.